Sources/IO/Core/HttpSceneLoader/index.js

```diff
@@ -301,8 +302,131 @@ function loadHttpDataSetReader(item, model, publicAPI) {
 
 // ----------------------------------------------------------------------------
 
+function loadHttpDataSetSeriesReader(item, model, publicAPI) {
```

Comment:
Can you try to factorize some of the duplicated code between `loadHttpDataSetReader` and this ?